install/ensure-correct-permissions-profiles-dir.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "${_group}Ensuring correct permissions on profiles directory ..."
+$dc run --rm vroom chown -R 1000:1000 /var/lib/sentry-profiles
```

Comment:
Pretty sure you need the following
```suggestion
$dc run --rm --entrypoint /bin/bash --user root vroom chown -R 1000:1000 /var/lib/sentry-profiles
```